test/src/locator.spec.ts

```diff
@@ -569,4 +569,20 @@ describe('Locator', function () {
       }
     });
   });
+
+  describe('Locator.prototype.wait', () => {
+    it('should work', async () => {
+      const {page} = await getTestState();
+      await page.setContent(`
```

Comment:
```suggestion
     page.setContent(`
```

to be sure that the div is not there